pkg/coastguard/controller/healthz.go

```diff
@@ -0,0 +1,16 @@
+package controller
+
+import (
+	"net/http"
+
+	"k8s.io/klog"
+)
+
+func serveHealthz(address string) {
+	http.HandleFunc("/healthz", func(w http.ResponseWriter, _ *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		_, _ = w.Write([]byte("OK"))
+	})
+
+	klog.Fatal(http.ListenAndServe(address, nil))
```

Comment:
Yes, because it means the listening address was used, and we cannot serve healthz, (used from kubernetes to detect health of the process), this  comes from federation or some other k8s example.